daskdev-sat/Dockerfile

```diff
@@ -9,13 +9,12 @@ RUN apt-get -qq update && \
     rm -rf /var/lib/apt/lists/*
 ENV SHELL /bin/bash
 
-RUN conda install -y dask=2.8.1  distributed
+RUN conda install -y -c conda-forge distributed tornado kubernetes dask-kubernetes python-kubernetes
 RUN conda install -y -c https://conda.saturncloud.io/pkgs sutils=0.5.1
-RUN conda install -y -c conda-forge kubernetes=1.13.5 dask-kubernetes=0.10.0 python-kubernetes=9.0
```

Comment:
I think we may actually want to keep the versions called out, here - if we don't, that'll mean that upstream changes could get pulled in without us being aware of them until users complain. On the flip side, though, I don't think it's really a big issue - thoughts @hhuuggoo ?

Reply:
Yeah we probably want to pin. I was just floating these to update them.